pkg/autodiscover/autodiscover_operators.go

```diff
@@ -39,6 +44,34 @@ const (
 	istioDeploymentName = "istiod"
 )
 
+type PackageManifest struct {
```

Comment:
Since we're already importing the operator-lifecycle-manager repo, you could use the official type for the package manifest [here](https://github.com/operator-framework/operator-lifecycle-manager/blob/b3926787314dfbc750d84b83c754510b30d78596/pkg/package-server/apis/operators/v1/packagemanifest_types.go#L22). Maybe there's also a client/schema to be used to avoid the need of the dynamic client.

Reply:
Cool Thx! I was looking for this but could not find it. I will update

Reply:
Fixed